float64 time

float64 dx
float64 dy
float64 dz

float64 dqx
float64 dqy
float64 dqz
float64 dqw
